fluent/syntax/ast.py

```diff
@@ -73,7 +73,7 @@ def visit(value):
 
         return fun(node)
 
-    def equals(self, other, with_spans=False):
+    def equals(self, other, ignored_fields=['span']):
```

Comment:
One more thing: I'd like to allow `a.equals(b, ignored_fields=None)` which won't be possible with the `if … is None` check.